run.sh

```diff
@@ -1,9 +1,26 @@
 #!/bin/bash
-python scripts/check_requirements.py requirements.txt
+
+# Check if python or python3 is installed
+if command -v python >/dev/null 2>&1; then
+    PY=python
+    PIP=pip
+elif command -v python3 >/dev/null 2>&1; then
+    PY=python3
+    PIP=pip3
+else
+    echo "Python is not installed. Please install Python 2.7 or 3.x."
```

Comment:
we only support python 3.10 and greater.

Reply:
Ah, thanks. Will close the PR.